### ISSUE TITLE
How to disable auto play

### ISSUE THREAD

**Issue by j3p0n** [eneim/Toro#52]
How to disable auto play, but user can still play/pause manualy? or disable auto play when network is on GPRS, and enabl auto play when on WIFI or 4G?

**Comment by eneim** [comment]
@j3p0n You can have your own *Strategy* which do nothing, for example a *lazyStrategy*. So when you want to disable auto playback, just call *Toro.setStrategy(lazyStrategy)*. See how I implement *Toro.rest(boolean)* for more information.

**Comment by j-a-s-o-n** [comment]
I created own lazyStrategy and called Toro.setStrategy(lazyStrategy) playback was disabled, but how to pause/play video when user clicks on video?. In this case adapter.getPlayer() returns null.

**Comment by foodied** [comment]
This intention of this library is to solely autoplay videos one at a time which it does nicely. Although If we want to temporarily disable autoplay, inside the viewholder that extends `ToroPlayer`, there a method `play()`. Just add `playerView.getPlayer().setPlayWhenReady(false)` after the `exoplayerViewHelper.play()`.

**Comment by eneim** [comment]
@foodied You can always return a 'conditional false' from ``wantsToPlay`` (assuming that you are using Toro 3.x) ;). It will disable the auto play for a certain ToroPlayer. To disable that for all players, you can use ``PlayerSelector.NONE`` for the Container.

**Comment by foodied** [comment]
I did set the `container.setPlayerSelector(PlayerSelector.NONE)` but it seems that video is not loaded as it comes to the center. It just shows the shutter view.

**Comment by eneim** [comment]
@foodied Yep, it will not start loading (SimpleExoPlayer will not call 'prepare') if you don't have it starts. You can put a thumbnail view on top to hide the black screen though ... What is the behaviour you demand?

**Comment by foodied** [comment]
Well, I guess immediate playback without any buffering when I click on play.

**Comment by foodied** [comment]
`EventListeners` inside `Playable` is not public.

**Comment by eneim** [comment]
@foodied which version of Java you are compiling with? Java 7+ will allow ``EventListeners`` to be accessible by default (it is a class inside a public interface).

**Comment by yjin** [comment]
@eneim Should the `container.setPlayerSelector(PlayerSelector.NONE)` be called before the container is populated? Looks like `setPlayerSelecter` doesn't work when called after the container is already populated and scrolled.
